dist/releases/4.16/config.toml

```diff
@@ -400,6 +400,10 @@ files = [
 error = "ErrNotDynLinked"
 files = ["/usr/libexec/catatonit/catatonit"]
 
+[[payload.openshift-istio-kiali-operator-container.ignore]]
+error = "ErrNotDynLinked"
+files = ["/usr/libexec/catatonit/catatonit"]
```

Comment:
Note for other reviews and historical reference. The catatonic binary is built statically, hence the exception here.